### PR TITLE
Tom/DLS-6517 - CGT - Changing 'Her Majesty's Revenue and Customs' to "His Majesty's Revenue and Customs"

### PR DESCRIPTION
**Pull request by TomRafferty**

### CODE REVIEW

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala [hunk 1]
```diff
@@ -4054,9 +4054,10 @@ class MultipleDisposalsTriageControllerSpec
 
       "redirect to the check your answers page" when {
 
-        val taxYear = sample[TaxYear].copy(
-          startDateInclusive = LocalDate.of(today.getYear, 4, 6),
-          endDateExclusive = LocalDate.of(today.getYear + 1, 4, 6)
+        val taxYearStartDate = TimeUtils.taxYearStart(today)
+        val taxYear          = sample[TaxYear].copy(
+          startDateInclusive = LocalDate.of(taxYearStartDate.getYear, 4, 6),
+          endDateExclusive = LocalDate.of(taxYearStartDate.getYear + 1, 4, 6)
```
Comment (LAJW) [inline test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala:4060]: This doesn't look like a date change.

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala [hunk 2]
```diff
@@ -4157,9 +4158,10 @@ class MultipleDisposalsTriageControllerSpec
 
       "redirect to the amend return disposaldate different taxyear page" when {
 
-        val taxYear = sample[TaxYear].copy(
-          startDateInclusive = LocalDate.of(today.getYear, 4, 6),
-          endDateExclusive = LocalDate.of(today.getYear + 1, 4, 6)
+        val taxYearStartDate = TimeUtils.taxYearStart(today)
+        val taxYear          = sample[TaxYear].copy(
+          startDateInclusive = LocalDate.of(taxYearStartDate.getYear, 4, 6),
+          endDateExclusive = LocalDate.of(taxYearStartDate.getYear + 1, 4, 6)
```
Comment (LAJW) [inline test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala:4164]: Also nestled in a date change commit.

**Review by LAJW — DISMISSED** [review]
Could you make sure date changes and code changes are not committed together?

**Comment by TomRafferty** [comment]
Ram's work has been moved under my name after I separated the copyright commits but his signature is still in the commit name and he should still be credited in git blame

apologies @rambabu-posa for the inconvenience

File: test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala (hunk 1 above)
Comment (TomRafferty) [reply test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala:4060]: this should now be resolved :)

**Review by LAJW — APPROVED** [review]

**Pull request merged**